source/Manila/usage.rst

```diff
@@ -447,9 +451,14 @@ The kernel client requires the ``ceph-common`` package to be installed. To acces
 the ``mount`` command of the form:
 
 .. code-block:: bash 
+    
+    # breakdown of the command 
 
-    mount -t ceph {mon1 ip addr}:6789,{mon2 ip addr}:6789,{mon3 ip addr}:6789:/{mount-point} -o name={access-id},secret={access-key}
+    mount -t ceph {mon1 ip addr}:6789,{mon2 ip addr}:6789,{mon3 ip addr}:6789:/{mount-point}  # export path for the share
+    
+    -o name={access-rule-name},secret={access-key} # name of the access rule and the secret associated to it
 
+    {mount-path} # path you want to attach the share to
```

Comment:
Whilst we're here, could we add a section about `chown` the directory to the user that's expected to access it:

```
The directory will be owned by `root` by default. To allow another user to read and write add the appropriate user and group like so:

.. code-block:: bash

  chown user:user {mount-path}
```